tools/release/03_publish_crates.ts

```diff
@@ -15,34 +14,12 @@ const dependencyCrates = getCratesPublishOrder(
 
 try {
   for (const [i, crate] of dependencyCrates.entries()) {
-    await publishCrate(crate);
-    console.log(`Finished ${i + 1} of ${dependencyCrates.length} crates.`);
+    await crate.publish();
+    $.log(`Finished ${i + 1} of ${dependencyCrates.length} crates.`);
   }
 
-  await publishCrate(cliCrate);
+  await cliCrate.publish();
 } finally {
   // system beep to notify error or completion
   console.log("\x07");
 }
-
-async function publishCrate(crate: Crate) {
-  if (isReal) {
-    await crate.publish();
-  } else {
-    await crate.publishDryRun();
```

Comment:
Unfortunately the dry publish didn't really work because you can't try dry publishing a crate that has dependencies that aren't published... so I removed this code.